test/unit/features/directives/local.spec.js

```diff
@@ -0,0 +1,120 @@
+import Vue from 'vue'
+
+describe('Directive v-local', () => {
+  it('should work', () => {
+    const vm = new Vue({
+      template: '<div><span v-local="v as foo.bar">{{v.baz}}</span></div>',
```

Comment:
I'd rather see the other way around `foo.bar as v` or a different _more js friendly_ syntax:

```html
<div v-local:v="foo.bar">
```

At the end, it would still be valid js. It can even be aliased to something like `<div $v="foo.bar"`

Reply:
Those are my thoughts on the feature. That being said, I feel that this crosses the line of Javascript/Magic even though I see when it can be comfortable

Reply:
updated